src/mme-app/conf/mme.json

```diff
@@ -24,6 +24,19 @@
 			"plmn1": "mcc=315,mnc=010",
 			"plmn2": "mcc=208,mnc=01"
         },
+	"dns":{
```

Comment:
Can you pleae write .md file about how to use this config ? or if not then pass on text description, I can help you in writing mark down file. We want to make sure our config has help.